handbook/engineering/distribution/index.md

```diff
@@ -100,7 +100,6 @@ Go, Docker, Kubernetes
 
 - We're hiring a [Product Manager](../../product/roles/product_manager.md) for this role. The engineering team is owning PM responsibilities in the meantime.
 - [Gonzalo Peci](../../../company/team/index.md#gonzalo-peci-hehim) ([Engineering Manager](../roles.md#engineering-manager))
-  - [Stephen Gutekanst](../../../company/team/index.md#stephen-gutekanst)
```

Comment:
I think your name should still be somewhere so we can document who your manager is and roughly what your plan is. For example: https://github.com/sourcegraph/about/pull/1248 (you can link to the doc you created)

Reply:
https://github.com/sourcegraph/about/pull/1754/files#r518274190